sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -1549,6 +1552,70 @@ aditof::Status CameraItof::parseJsonFileContent() {
     return status;
 }
 
+aditof::Status
+CameraItof::saveDepthParamsToJsonFile(const std::string &savePathFile) {
+    using namespace aditof;
+    Status status = Status::OK;
+
+    cJSON *rootjson = cJSON_CreateObject();
+    cJSON *depthParamjson = cJSON_CreateObject();
+    cJSON_AddItemToObject(rootjson, "depthParams", depthParamjson);
+
+    for (auto pfile = m_ini_depth_map.begin(); pfile != m_ini_depth_map.end();
+         pfile++) {
+        size_t lastUnderscorePos = pfile->second.find_last_of("_");
+        if (lastUnderscorePos == std::string::npos) {
+            continue;
+        }
+        size_t dotPos = pfile->second.find_last_of(".");
+        std::string modeName = pfile->second.substr(
```

Comment:
I believe that m_ini_depth_map already contains the name of the mode (it's the key and can be obtained with pfile->first). So there is no need to extract it again from the file name.

Reply:
Yes indeed, pfile->first contain the mode name.